microcosm_flask/swagger/definitions.py

```diff
@@ -253,6 +259,10 @@ def add_responses(swagger_operation, operation, ns, func):
         if not hasattr(swagger_operation, "produces"):
             swagger_operation.produces = []
         swagger_operation.produces.append(response_resource)
+    elif not response_resource:
```

Comment:
I had to add some customization for 200 vs 204 responses. In order to allow the success case of an operation to have a variable status code, I can't just use `operation.value.default_code`. Rather than generalize the operation to code mapping with some new function, I'm just handling the 204 case.